### DISCUSSION TITLE
How to watch select field text using useWatch?

### DISCUSSION THREAD

**Discussion by abdulmateentechbits** [react-hook-form/react-hook-form#9186] [Q&A]
How can i watch select field text rather then value,
I want to watch the item1,
`<select>
<option value="1">,item1</option>
</select>`

**Comment by fahadsohail482** [comment]
@abdulmateentechbits I am not sure if that's possible using useWatch api. useWatch returns the values of the fields. 
One suggestion is to use the same name for text and value of select.
